.github/workflows/build.yml

```diff
@@ -28,33 +22,35 @@ jobs:
       - name: Update Fastlane
         run: bundle update fastlane
```

Comment:
you shouldn't need to update fastlane here, this is handled automatically by dependabot or renovate

Reply:
thanks @frett , I will remove this step